core/events/services/classic/bootstrap.py

```diff
@@ -0,0 +1,151 @@
+"""Generate synthetic data for testing and development purposes."""
+from typing import List
+import random
+from datetime import datetime
+from mimesis import Person, Internet, Datetime
+from mimesis import config as mimesis_config
+
+from arxiv import taxonomy
+from . import models
+
+LOCALES = list(mimesis_config.SUPPORTED_LOCALES.keys())
+
+def _get_locale() -> str:
+    return LOCALES[random.randint(0, len(LOCALES) - 1)]
+
+
+def _epoch(t: datetime) -> int:
+    return int((t - datetime.utcfromtimestamp(0)).total_seconds())
+
+
+LICENSES = [
```

Comment:
We should consider adding this to arxiv-base in the future.

Reply:
Agreed